cmd/virt-launcher/node-labeller/node-labeller.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+arch=$(uname -m)
+if [ "$arch" != "x86_64" ]; then
+  echo "Not running on x86 architecture"
```

Comment:
Should it be echo "Not running on ${arch} architecture" ?

Reply:
No, the logic intent is that if the arch is detected as anything other than x86_64, the node-labeler should exit.

Reply:
> No, the logic intent is that if the arch is detected as anything other than x86_64, the node-labeler should exit.

Yes, I get your idea, but does the sentence, `Not running on x86 architecture`, means this script does not running on X86?

Reply:
That is correct, yes. I could change this to read:

`"Not running since architecture: ${arch} is not x86_64"`

if we're wanting to go for something a bit more explicit

Reply:
I think this change is no longer neede. This was handled by a different PR in the meantime. Check https://github.com/kubevirt/kubevirt/pull/8293/files#diff-eef8842fa63488df8f872df6cef23d52bc8d84bce9180aff84ff0521845ef021R6 above.